spaces/S000104/README.md

```diff
@@ -11,12 +11,7 @@ refs:
 ---
 
 
-The product topology on the space of ordinals $\omega_1$ crossed with the
-product space $I^I$.
+The product topology on {S35} crossed with {S103}.
```

Comment:
Maybe it would be clearer to keep the old symbols in there.  Up to you.

```suggestion
The product topology on $\omega_1$ ({S35}) crossed with $I^I$ ({S103}).
```